### ISSUE TITLE
Load .cue file when opening large wav/flac/ape directly. 

### ISSUE THREAD

**Issue by dikonov** [DeaDBeeF-Player/deadbeef#2131]
When the user opens a wave/flac/ape/.. file directly older versions of deadbeef used to load a cue file with matching name when available. This is very convenient for flac+cue disk images.

The new version 1.8 does not do it and requires to open the cue instead of the flac to see all tracks inside. Some cue files have wrong file paths inside. 

Please, add back (at least as an option) the autoloading of <same filename>.cue on playing audio files.   

Deadbeef version: 1.8.0
OS: Linux

**Comment by Alexey-Yakovenko** [comment]
It's impossible to add back, since this would break these use cases:
1. opening cuesheets referencing multiple audio files
2. opening image files without splitting them by cuesheet

Wrong paths are auto-corrected when adding the folder containing image+cue.
Please let me know if there's a specific use case which you can't solve, and I will see what can be done.

**Comment by dikonov** [comment]
WRONG

Autoloading a cuesheet will not break point 1, if the player will either 
 - load cue and ignore FILE sections that point to othe files. 
    (This will help to load metadata stored in .cue)
- forget about the original file and load .cue as if the user clicked the cuesheet.

Opening files that have a cuesheet without splitting is a valid but comparatively rare task. It can be done with a commandline option. It is quite possible to have alternative actions in file managers too by using e.g. deadbeef --nocue %p or deadbeef --loadcue %p.

Current state makes deadbeef 1.8 significanly less intuitive and less convenient.

**Comment by Alexey-Yakovenko** [comment]
Not really. Many people use deadbeef and foobar2000 interchangeably, and want the same cuesheet handling behavior, which is what I'm trying to achieve -- both from UI standpoint, and from features standpoint.

Treating cue as metadata files worked well indeed, but was blocking support of cuesheets with multiple FILE entries, and was confusing hell of a lot of users, so a special section had to be added to the help file. Most users do expect to open cue files instead of audio, when they want the splitting.

The current behavior is way less ambiguous.

**Comment by dikonov** [comment]
I am a proud DEADBEEF user! _and I do not give a damn about foobar_. I have my habits formed by deadbeef and I would like to keep it this way. 

There may be a compromise solution of having 1) a setting that would control the default behavior (autoload CUEsheets or not) 2) a commandline switch with the same effect, which would give us possibility to use it in the default command issued by the file managers upon clicking an audio file.

**Comment by Alexey-Yakovenko** [comment]
I can understand your opinion, but you should also understand what deadbeef is -- it's basically an opensource foobar2000 clone. It aims to have a very close UI/UX to foobar2000.

An option for autodetecting cuesheets can be re-implemented somehow, but currently there's no code that would do that. It's not as simple as toggling an option. This autodetection would obviously not work with tracks+cue / multiple-images+cue albums either.

So such feature looks very much like a step back to me. But anyway, if you want it -- I can keep the case open.

**Comment by icher** [comment]
> Please let me know if there's a specific use case which you can't solve, and I will see what can be done.

For now you can't open folder with image+cue (ape+cue for example), deadbeef load only image file without cue. So, if you have few albums like this, you should open EVERY folder and select cue file. REALLY annoying. Please do something with it. Maybe new menu item like `Add folder with image+cue`, if you don't want change default (or foobar2000 compatible) behavior. 

> This autodetection would obviously not work with tracks+cue / multiple-images+cue albums either.

Well, for tracks cue is optional in most cases. They have correct tags inside. I think processing simple cases like `1 image 1 cue per folder` would be just fine. In complicated cases users can select cue manually.

p.s. I don't care about foobar2000 too.

**Comment by Alexey-Yakovenko** [comment]
> For now you can't open folder with image+cue (ape+cue for example), deadbeef load only image file without cue. So, if you have few albums like this, you should open EVERY folder and select cue file. REALLY annoying.

I just tried opening a folder with ape+cue, and it works just fine, it loads correctly, both by drag-dropping a folder into playlist, and by clicking Add folder(s), and selecting one.

**Comment by Alexey-Yakovenko** [comment]
> Well, for tracks cue is optional in most cases. They have correct tags inside. 

It's primarily about multiple images + single cue in the same folder. I agree, that the regular tracks+cue usually don't need the cuesheet at all.

**Comment by icher** [comment]
> I just tried opening a folder with ape+cue, and it works just fine, it loads correctly, both by drag-dropping a folder into playlist, and by clicking Add folder(s), and selecting one.

Are you sure? Cause I have really different experience:

https://user-images.githubusercontent.com/8604717/110088277-dd00af80-7dbe-11eb-801b-33e78de5cf7e.mp4

Fedora Linux 33
Gnome 3.38.3
Wayland
deadbeef-static_1.8.7-1_x86_64

**Comment by Alexey-Yakovenko** [comment]
Ok, good to know. Would you be so kind to post a bug about it?

**Comment by icher** [comment]
> Ok, good to know. Would you be so kind to post a bug about it?

#2548
